specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/createSubscriptionUnderBillingProfile.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "billingAccountName": "12a6eed9-5d39-4103-a5db-2db1db0a8c67",
+	"billingProfileName": "27VR-HDWX-BG7-TGB",
```

Comment:
Should be billingProfileName parameter be 'dbcb81c8-cda9-4b01-b9fc-831e6a2e152e' as defined in the response payload?

Reply:
Yes, thank you. Should be the same one from request payload.